### PR TITLE
ICDS - add error into pdf file

### PR DESCRIPTION
**Pull request by lwyszomi**
Hi @czue ,

This change should help and is only to figure out why PDF export don't work on the production. With data which I have I can't reproduce issue with exporting. 

This should be deployed only on India server.

Please let me know if you have any questions.

Regards,
Łukasz

### CODE REVIEW

**Review by czue — APPROVED** [review]

**Comment by czue** [comment]
> This should be deployed only on India server.

@lwyszomi you mean that it is only urgently needed on india server, but is fine to merge/deploy everywhere, right?

Assuming yes - @esoergel can you make sure this gets merged and deployed to india today?

**Comment by lwyszomi** [comment]
@czue @esoergel yes

File: custom/icds_reports/utils.py [hunk 1]
```diff
@@ -439,16 +439,22 @@ def zip_folder(pdf_files):
...
     return zip_hash
 
 
 def create_pdf_file(pdf_hash, pdf_context):
     template = get_template("icds_reports/icds_app/pdf/issnip_monthly_register.html")
     resultFile = cStringIO()
     client = get_redis_client()
+    try:
+        pdf_page = template.render(pdf_context)
+    except Exception as ex:
```
Comment (esoergel) [inline custom/icds_reports/utils.py]: Are you pretty confident this is where the exception occurs? I generally think of django template rendering as pretty fault tolerant.  My guess would be that errors would occur either during the call to `CreatePDF` or when assembling the `pdf_context`.  You can also use `notify_exception` to assure that we get a traceback with relevant info, although I'm not sure if you have direct access to that.

Comment (lwyszomi) [reply custom/icds_reports/utils.py]: @esoergel I am 100% sure that the problem is in evaluating ```pdf_context``, when we try to access to specific fields. From functionality context, we do this only in render function that's why I added try/except block in this lines.

Comment (esoergel) [reply custom/icds_reports/utils.py]: Makes sense, thanks for clarifying!

**Pull request merged**